firmware/Inc/main.h

```diff
@@ -99,6 +101,22 @@
 
 int CMain(void);
 void Error_Handler();
+
+ADC_HandleTypeDef* getADC1Handle(void);
+ADC_HandleTypeDef* getADC2Handle(void);
+ADC_HandleTypeDef* getADC3Handle(void);
+DMA_HandleTypeDef* getDMAADC1Handle(void);
+DMA_HandleTypeDef* getDMAADC2Handle(void);
+DMA_HandleTypeDef* getDMAADC3Handle(void);
+
+I2C_HandleTypeDef* getI2C1Handle(void);
+I2C_HandleTypeDef* getI2C2Handle(void);
+
+TIM_HandleTypeDef* getTIM1Handle(void);
+TIM_HandleTypeDef* getTIM2Handle(void);
+TIM_HandleTypeDef* getTIM3Handle(void);
+TIM_HandleTypeDef* getTIM4Handle(void);
```

Comment:
Are these necessary? Since the handles are statically defined and populated in main.c, why not just move the declaration of the handles to main.h?